Dockerfile

```diff
@@ -15,10 +15,14 @@ RUN gem install bundler
 RUN bundle config set deployment 'true'
 RUN bundle config set without 'development test'
 
-RUN mkdir /usr/src/app
-COPY . /usr/src/app
 WORKDIR /usr/src/app
+
+COPY Gemfile Gemfile.lock ./
 RUN bundle install
+
+COPY package.json yarn.lock ./
 RUN yarn install
 
+COPY . ./
+
```

Comment:
@ErikSchierboom For this change, does this not only matter if we cache the intermidiate dockerfile stages somewhere. Atm I think they're lost every time. So should we cache these on GitHub's cache somehow?

Reply:
> Atm I think they're lost every time. So should we cache these on GitHub's cache somehow?

Ah yes, we should. There was mention that GitHub would be implementing this themselves, but they haven't done that so far. There is a bit of complexity involved. Want to do this at a later stage?